pkg/internal/netolly/transform/k8s/kubernetes.go

```diff
@@ -0,0 +1,141 @@
+// Copyright Red Hat / IBM
+// Copyright Grafana Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This implementation is a derivation of the code in
+// https://github.com/netobserv/netobserv-ebpf-agent/tree/release-1.4
+
+package k8s
```

Comment:
Just wondering, did you by any chance fix the headless services issue with the  k8s decoration we hit during our hackathon? I remember that the jaeger service or some other one from the OTel k8s demo caused us to crash. We fixed it by assigning it an ip.

Reply:
This code is taken directly from the hackathon, so I'd say so but I'll double check.